CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
 # UNRELEASED
 
+-   Update matrix-rusk-sdk to `35a2ce9`, which includes:
+
+    -   Add variants for plain text and encrypted to-device events ([#4935](https://github.com/matrix-org/matrix-rust-sdk/pull/4935))
```

Comment:
Does this affect the API exposed by crypto-wasm? If so, it would be good to say how. If not, it would probably be good to say so explicitly.